includes/user-functions.php

```diff
@@ -38,7 +37,14 @@ function edd_get_users_purchases( $user = 0, $number = 20, $pagination = false,
 		return false;
 	}
 
-	$status = $status === 'complete' ? 'publish' : $status;
+	if ( ! is_array( $status ) && is_string( $status ) ) {
```

Comment:
Only reason I say this is that many functions in WP Core allow an array, or comma separated list of statuses. Just makes it consistant and doesn't take much on our end.